sass/ui-kit.scss

```diff
@@ -3,3 +3,4 @@
 @import "../ui-kit/assets/sass/ui-kit.scss";
 @import "components/*";
 @import "modules/*";
+@import "vendor/font-awesome/font-awesome";
```

Comment:
@mgdhs done! I didn't think that would work since the documentation for glob seems to suggest it won't recursively do sub directories under vendor, just files.